src/xenia/gpu/shaders/primitive_point_list.gs.hlsl

```diff
@@ -19,7 +19,7 @@ void main(point XeVertexPreGS xe_in[1],
   // Shader header writes -1.0f to point_size by default, so any positive value
   // means that it was overwritten by the translated vertex shader.
   float2 point_size =
-      xe_in[0].post_gs.pre_ps.point_params.z > 0.0f
+      xe_in[0].post_gs.pre_ps.point_params.z >= 0.0f
```

Comment:
Not sure if point size 0.0f should be dropped or drawn with min size.
Makes no noticeable difference in Fable 2.